gto-support-eventbus/src/main/java/org/ccci/gto/android/common/eventbus/content/EventBusLoaderHelper.java

```diff
@@ -30,41 +33,45 @@ public EventBusLoaderHelper(@NonNull final Loader loader, @Nullable final Object
         }
     }
 
-    public void setEventBusListener(@Nullable Object listener) {
-        if(listener == null) {
-            listener = mLoader;
-        }
-
-        synchronized (this) {
-            // Register the new listener and unregister the old listener.
-            // We overlap registrations to not drop broadcasts
-            if (mLoader.isStarted()) {
-                registerListener(listener);
-                unregisterListener(mEventBusListener);
-            }
+    public synchronized void addEventBusSubscriber(@NonNull final EventBusSubscriber subscriber) {
+        mSubscribers.add(subscriber);
+        registerSubscriber(subscriber);
+    }
 
-            // save new listener
-            mEventBusListener = listener;
-        }
+    public synchronized void removeEventBusSubscriber(@NonNull final EventBusSubscriber subscriber) {
+        mSubscribers.remove(subscriber);
+        unregisterSubscriber(subscriber);
     }
 
     public void onStartLoading() {
         synchronized (this) {
-            registerListener(mEventBusListener);
+            for (final EventBusSubscriber subscriber : mSubscribers) {
+                registerSubscriber(subscriber);
+            }
         }
     }
 
-    public void unregister() {
-        synchronized (this) {
-            unregisterListener(mEventBusListener);
+    public void onAbandon() {
+        unregisterSubscribers();
+    }
+
+    public void onReset() {
+        unregisterSubscribers();
+    }
+
+    private synchronized void unregisterSubscribers() {
+        for (final EventBusSubscriber subscriber : mSubscribers) {
+            unregisterSubscriber(subscriber);
```

Comment:
Sorry, for some reason I was mind blanking and thinking it was calling `removeEventBusSubscriber` instead of `unregisterSubscriber`.